src/transformState.js

```diff
@@ -4,8 +4,23 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
You can use `switch` here
```suggestion
   for (const action of actions) {
    switch (action.type) {
    case 'addProperties':
```